pages/mesosphere/dcos/services/data-science-engine/1.0.1/dynamic-allocation/index.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: layout.pug
+navigationTitle: Dynamic Allocation 
+excerpt: Enabling Dynamic Allocation with Shuffle Service for DC/OS Data Science Engine Spark
+title: Dynamic Allocation
+menuWeight: 7
+enterprise: true
+model: /mesosphere/dcos/services/data-science-engine/data.yml
+render: mustache
+---
+
+# Prerequisites
+
+- {{ model.techName }} must use a host networking mode, due to the specifics of Spark executors communication with Shuffle Service and the way shuffle blocks are served. 
+
+  Once Executor has written a shuffle block to disk, it registers the block with the driver and the block location includes the host IP address. The block is then fetched by other executors using this IP. Using a virtual network does not fit in this architecture because each executor will have a unique IP address not representing the physical host where the blocks are located.
+- Shuffle Service must be installed on every host where Spark Executors will be running. 
+  
+  Both Shuffle Service and Spark Executors should use the same shared host folder/volume for saving and serving shuffle blocks.
+- The Spark user (Linux user running the Shuffle Service and Executors) should have read-write privileges for the shared folder.
```

Comment:
Are formatting and paragraphs ok in this section?

Reply:
Yes. If you do a local build, you will see that each introductory sentence is followed by an indented paragraph which explains the subject in more detail. I did this to break up the lumps of expository text. See screenshot below to see what the finished version looks like:

<img width="622" alt="Screen Shot 2020-01-24 at 11 04 29 AM" src="https://user-images.githubusercontent.com/37596413/73096314-54180480-3e99-11ea-9f60-c2e16c6e2f0b.png">